Modules/ManualAssessment/classes/class.ilObjManualAssessmentGUI.php

```diff
@@ -266,4 +268,18 @@ protected function afterSave(ilObject $a_new_object) {
 		$this->ctrl->setParameter($this, "ref_id", $a_new_object->getRefId());
 		ilUtil::redirect($this->ctrl->getLinkTargetByClass('ilmanualassessmentsettingsgui', 'edit', '', false, false));
 	}
+
+	public function addToNavigationHistory(){
+		global $DIC;
+		$ilNavigationHistory = $DIC['ilNavigationHistory'];
+		
+		if(!$this->getCreationMode() &&
+			$this->ilAccess->checkAccess('read', '', $_GET['ref_id']))
+		{
```

Comment:
bitte hier
$access_handler = $this->object->accessHandler();
$access_handler->checkAccessToObj($this->object,'read')
benutzen. Siehe auch die getTabs methode.